sematic/log_reader.py

```diff
@@ -57,16 +58,21 @@ def get_resolution(self, resolution_id: str) -> Resolution:
     def get_run(self, run_id: str) -> Run:
         return self.value[0].get_run(run_id)
 
+    def get_jobs_by_run_id(self, run_id: str) -> List[Job]:
+        return self.value[0].get_jobs_by_run_id(run_id)
 
-def log_prefix(run_id: str, job_type: JobType):
+
+def log_prefix(run_id: str, job_kind: JobKindString):
+    log_kind = "worker" if job_kind == JobKind.run else "driver"
     return LOG_PATH_FORMAT.format(
-        prefix=V2_LOG_PREFIX, run_id=run_id, log_kind=job_type.value
+        prefix=V2_LOG_PREFIX, run_id=run_id, log_kind=log_kind
```

Comment:
What will be the consequences of this inconsistency?
Time for V3 instead? Or do not change `JobType` to `JobKind`?

Reply:
Added this:
```
    # TODO: move log writing to use "run"/"resolver" instead
    # of "worker"/"driver". Wait to do it until we can add in
    # some backwards compatibility so we can still read logs
    # written by clients that haven't yet upgraded. Probably
    # wait until after deprecating V1, so we don't have to support
    # *3* formats in this module at one time.
```